yarn-project/archiver/src/archiver/archiver.ts

```diff
@@ -269,7 +268,7 @@ export class Archiver implements ArchiveSource {
     // create the block number -> block hash mapping to ensure we retrieve the appropriate events
     const blockHashMapping: { [key: number]: Buffer | undefined } = {};
     retrievedBlocks.retrievedData.forEach((block: L2Block) => {
-      blockHashMapping[block.number] = block.getCalldataHash();
+      blockHashMapping[block.number] = block.body.getCalldataHash();
```

Comment:
Everywhere in the codebase block hash is a hash of header so the name here is confusing. Would rename it to something like bodyHashes.